community/modules/scheduler/schedmd-slurm-gcp-v6-controller/scripts/cleanup_compute.sh

```diff
@@ -32,25 +32,25 @@ fi
 echo "Deleting compute nodes"
 node_filter="labels.slurm_cluster_name=${cluster_name} AND labels.slurm_instance_role=compute"
```

Comment:
1. I think a `"NOT status:(STOPPING TERMINATED)"` filter would go a long way toward addressing the real cause of this problem.
2. I like the idea of retrying failures but not forever. I think something like

```
max_failures=5
retry_attempt=0
gcloud compute instances delete || [ $((retry_attempt++)) -gt "$max_failures"]]
```